victorLi/assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js

```diff
@@ -0,0 +1,77 @@
+// Problem Statement #
+
+// Given a binary tree and a node, find the level order successor of the given node in the tree. The level order successor is the node that appears right after the given node in the level order traversal.
+
+//            (  1  )
+//           /       \
+//       (  2  )    (  3  )
+//       /     \       \
+// (  4  )  (  5  )   (  6  )
+
+// Given Node: 3
+// Level Order Successor: 4
+
+
+//             (  12  )
+//            /       \
+//       (  7  )    (  1  )
+//       /            /    \
+//  (  9  )       (  10  )(  5  )
+
+// Given Node: 9
+// Level Order Successor: 10
+
+
+/*
+  Traverse tree using BFS
+  If current node = target
+  return the first item on the queue
+*/
+
```

Comment:
Would recommend a more detailed overview that includes notes about input, expected output, edge cases, etc.